python/src/FunctionalChaosAlgorithm_doc.i.in

```diff
@@ -19,167 +19,191 @@ Available constructors:
 Parameters
 ----------
 inputSample, outputSample : 2-d sequence of float
-    The input random variables :math:`\vect{X}=(X_1, \dots, X_{n_X})^T`
-    and the output samples :math:`\vect{Y}` of a model evaluated apart.
+    Sample of the input - output random vectors
 model : :class:`~openturns.NumericalMathFunction`
-    Physical model :math:`g` such as :math:`\vect{Y} = g(\vect{X})` that
-    behaves as a :class:`~openturns.NumericalMathFunction`. When
-    :math:`\vect{Y}` is of dimension :math:`n_Y>1`, OpenTURNS operates marginal
-    by marginal, using the same multivariate orthonormal basis
-    :math:`(\Psi_k(\vect{x}))_{k \in \Nset^{n_X}}` for all the marginals.
+    Model :math:`g` such as :math:`\vect{Y} = g(\vect{X})`.
 distribution : :class:`~openturns.Distribution`
-    Joint probability density function :math:`f_{\vect{X}}(\vect{x})`
-    of the physical input vector :math:`\vect{X}`.
+    Distribution of the random vector :math:`\vect{X}`
 adaptiveStrategy : :class:`~openturns.AdaptiveStrategy`
     Strategy of selection of the different terms of the multivariate basis.
 projectionStrategy : :class:`~openturns.ProjectionStrategy`
-    Defines the strategy of projection of the empirical model results in the
-    selected orthonormal basis. If it is not mentioned, the projection
-    strategy considered is the :class:`~openturns.LeastSquaresStrategy`.
+    Strategy of evaluation of the coefficients :math:`\alpha_k`
 weights : sequence of float
-    Weights of each point of the *inputSample*. The weights :math:`\omega_i`
-    are determined such that
-    :math:`\sum_{i\in I}\omega_i \delta_{\vect{X}_i} \simeq p_{\vect{X}}`,
-    where :math:`p_{\vect{X}}` is the distribution of the input random vector
-    :math:`\vect{X}`. When not specified, the *weights* are all equal to:
-    :math:`\omega_i = \frac{1}{cardI}`
+    Weights :math:`\omega_i` associated to the data base
+
+    Default values are :math:`\omega_i = \frac{1}{N}` where 
+    *N=inputSample.getSize()*
+
+See also
+--------
+FunctionalChaosResult
 
 Notes
 -----
-Consider a physical model :math:`g` depending
-on a set random variables :math:`\vect{X}=(X_1, \dots, X_{n_X})^T`.
-We call functional chaos expansion the class of spectral methods which gathers
-all types of response surface that can be seen as a projection of the physical
-model in an orthonormal basis. This class of methods uses the Hilbertian space
-(square-integrable space: :math:`L^2`) to construct the response surface.
+Consider :math:`\vect{Y} = g(\vect{X})` with :math:`g: \Rset^d \rightarrow \Rset^p`, 
+:math:`\vect{X} \sim \cL_{\vect{X}}` and :math:`\vect{Y}` with finite variance: 
+:math:`g\in L_{\cL_{\vect{X}}}^2(\Rset^d, \Rset^p)`.
+
+When  :math:`p>1`, the functional chas algorithm is used on each marginal 
```

Comment:
When  :math:`p>1`, the functional chas --> When :math:`p>1`, the functional chaos